File: backend/src/main/java/ma/dnaengineering/backend/entities/Employee.java
package ma.dnaengineering.backend.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "Employee")
public class Employee {
    @Id
    private int id;
    private String employee_name;
    private String job_title;
    private double salary;
}

File: frontend/src/components/Employee.js
const Employee = ({ employees }) => {
    return (
        <table>
            <thead>
            <tr>
                <th>Id</th>
                <th>Name</th>
                <th>Job Title</th>
                <th>Salary</th>
            </tr>
            </thead>
            <tbody>
            {employees && employees.map((employee) => (
                <tr key={employee.id}>
                    <td>{employee.id}</td>
                    <td>{employee.employee_name}</td>
                    <td>{employee.job_title}</td>
                    <td>{employee.salary}</td>
                </tr>
            ))}
            </tbody>
        </table>
    );
};

export default Employee;